.cirrus.yml

```diff
@@ -0,0 +1,20 @@
+# Adapted from
+# https://github.com/benfred/py-spy/blob/d79a69a4f6f380a179944106126ce88265c4bab1/.cirrus.yml
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+
+freebsd_task:
+  name: $TOOLCHAIN x86_64-unknown-freebsd
+  env:
+    matrix:
+      - TOOLCHAIN: stable
+      - TOOLCHAIN: beta
+      - TOOLCHAIN: nightly
```

Comment:
I'd suggest just testing against the stable version - cirrus ci can be pretty slow to run builds, and running against stable/beta/nightly will take a bunch of extra time.